lms/djangoapps/courseware/management/commands/tests/test_dump_course.py

```diff
@@ -205,15 +204,6 @@ def check_export_file(self, tar_file):  # pylint: disable=missing-docstring
         assert_in('edX-simple-2012_Fall/sequential/Lecture_2.xml', names)
 
 
-class CommandsXMLTestCase(CommandsTestBase):
```

Comment:
It's not directly related to CCX. 
I made a change to the 'dump_course_ids' management command, which broke the tests for the 3 module store types. Fixing the one for XML did not seem straightforward.